_includes/module-loading/bundling-browserify-cjs-main.md

```diff
@@ -0,0 +1,37 @@
+1. Add the {{site.productname}} assets and configuration into the project and provide a target element to initialize the editor.
+
+    Example `src/main.js`
+
+    ```js
+    var editor = require('./editor.js');
+    /* Create a parent element for the textarea */
+    var parent = document.createElement('p');
+    /* Create a textarea with id="editor" for the TinyMCE `selector` option */
+    function editorArea() {
+      var element = document.createElement('textarea');
+      element.id = 'editor';
+      return element
+    }
+    /* Add elements to page */
+    parent.appendChild(editorArea());
```

Comment:
Fact check the textarea will be inside the p tag is that correct?

Reply:
Correct.